src/components/TableHead/TableHead.tsx

```diff
@@ -25,7 +25,7 @@ export interface TableHeadProps extends MuiTableHeadProps {
   dragRows?: boolean;
   selected: number;
   items: Node[];
-  toolbar: React.ReactNode | React.ReactNodeArray;
+  toolbar?: React.ReactNode | React.ReactNodeArray;
```

Comment:
There shouldn't be situations in which you can select multiple items and do nothing with them

Reply:
done